src/themes/theme.ts

```diff
@@ -31,6 +31,7 @@ export default createTheme({
     },
     shape: {
         borderRadius: '3px',
+        borderRadiusMedium: '8px',
```

Comment:
@NicolaeUnleash can we standardize border radius? Does 8px vs 10px make a difference in the UI?

Reply:
that's a good idea to make a standard, this is what i used to keep consistency, maybe we can create these 3 tokens(s,m,l):
-> 4px (for buttons, inputs)
-> 8px for (table header, dropdown)
-> 12px for bigger elements (bigger containers, dialogs)

not sure what we have now in the app, but it could be a start to align them